pkg/install/linux_openrc.go

```diff
@@ -28,6 +28,8 @@ command_args="{{range .Arguments}}{{.}} {{end}}"
 name=$(basename $(readlink -f $command))
 supervise_daemon_args="--stdout /var/log/${name}.log --stderr /var/log/${name}.err"
 
+rc_ulimit="${K0S_ULIMIT:--c unlimited -n 1048576 -u unlimited}"
```

Comment:
```suggestion
: "${rc_ulimit=-c unlimited -n 1048576 -u unlimited}"
```

This way user can override the setting in `/etc/conf.d/k0sworker` using `rc_ulimit`. It is also backwards compatible for users who already may have set `rc_ulimit` in their `/etc/conf.d/k0sworker`

Reply:
actually, I don't think we should set `-c unlimited` by default for production.

Reply:
core file size default to zero, but leave -c untouched is ok for me.